main.tf

```
@@ -528,6 +528,7 @@ module "lacework_ct_iam_role" {
   version                 = "~> 0.1"
```

This change actually depends on https://github.com/lacework/terraform-aws-iam-role/pull/32

We need to bump the min version to:
```suggestion
  version                 = "~> 0.3"
```